TypeSystem/flow-analysis/reachability_A02_t05.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion True literal: If `N` is the literal `true`, then:
+///  Let `true(N) = before(N)`.
+///  Let `false(N) = unreachable(before(N))`.
+///
+/// @description Checks reachability after `true` literal in collection literals
+/// @author sgrekhov22@gmail.com
+
+void test1() {
+  late int i;
```

Comment:
I think this one could be non-late. Same situation below.

Reply:
Of course. Thank you for noticing! Fixed.